src/object_store/src/object/opendal_engine/opendal_object_store.rs

```diff
@@ -59,6 +62,22 @@ impl OpendalObjectStore {
     }
 }
 
+pub fn new_http_client(config: &ObjectStoreConfig) -> ObjectResult<HttpClient> {
+    let mut client_builder = reqwest::ClientBuilder::new();
+
+    if let Some(keepalive_ms) = config.s3.object_store_keepalive_ms.as_ref() {
+        client_builder = client_builder.tcp_keepalive(Duration::from_millis(*keepalive_ms));
+    }
+
+    if let Some(nodelay) = config.s3.object_store_nodelay.as_ref() {
+        client_builder = client_builder.tcp_nodelay(*nodelay);
+    }
+
+    client_builder = client_builder.https_only(false);
```

Comment:
`https_only` is default to `false` in `ClientBuilder`. There is no need to set it explicitly.